mlops/parallelm/mlops/mlops.py

```diff
@@ -932,6 +933,52 @@ def load_time_capture(self, input_file):
         from parallelm.mlops.time_capture.time_capture import TimeCapture
         return TimeCapture(input_file)
 
+    def attach(self,
+               mlapp_id,
+               mlops_server=Constants.MLOPS_DEFAULT_HOST,
+               mlops_port=Constants.MLOPS_DEFAULT_PORT,
+               user=Constants.MLOPS_DEFAULT_USER,
+               password=None):
+        """
+        Attach to a running ION and run in its context.
+        Side effect: sets up mlops_context
+        :param mlapp_id: the id of the ION to connect to
+        :param mlops_server: the host to connect to
+        :param mlops_port: the port MLOps is using
+        :param user: user name to use for connection
+        :param password: password to use for authentication, if None (default) then prompt for password
+        :return:
+    `
+        Note: Attach only works for pure python code
+        """
+        self._logger.info("Connecting to mlops: {} {}: {} user: {} pass: {}".format(
+            mlops_server, Constants.ION_LITERAL, mlapp_id, user, password))
+
+        # Connecting directly the server
+        rest_helper = MlOpsRestFactory().get_rest_helper(MLOpsMode.ATTACH, mlops_server, mlops_port, None)
+        token = rest_helper.login(user, password)
+
+        # Setting the environment for mlops
+        ci = ConfigInfo()
+        ci.token = token
+        ci.zk_host = None
+        ci.mlops_port = str(mlops_port)  # Constants.MLOPS_DEFAULT_PORT
+        ci.mlops_server = mlops_server
+        ci.ion_id = mlapp_id
+        ci.mlops_mode = MLOpsMode.ATTACH
+        ci.output_channel_type = OutputChannel.PYTHON
+
+        # TODO: for now assume node "0" - allow providing the node id or just become any node
+        ci.ion_node_id = "0"
```

Comment:
can we use some other value (@lior-amar any suggestions ?) since 0 is definitely already used? maybe pick a number greater than the last node in the mlapp? and add it to the constants file?